blocks/library/gallery/index.js

```diff
@@ -121,16 +124,24 @@ registerBlockType( 'core/gallery', {
 			controls,
 			focus && images.length > 1 && (
```

Comment:
Responded in core slack, I think this is a great place to show contextual help. We want to ensure to have best practices in place that suggest no more than 2 lines, but it feels like it's worth using this spot to show this.